ci/provision-certificate.sh

```diff
@@ -5,8 +5,8 @@ set -eu
 curl -L -o jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
 chmod +x ./jq
 
-pip install certbot==2.6.0
-pip install certbot-dns-route53==2.6.0
+pip install certbot==3.3.0
```

Comment:
While we're at it, should we remove the install of `jq` above? `jq` is included in the general-task image already using the latest version

Reply:
done